### DISCUSSION TITLE
Where do the pirates near Terminus come from?

### DISCUSSION THREAD

**Discussion by bene-dictator** [endless-sky/endless-sky#7358] [General]
While I was mining platinum in Orbona with a Headhunter, I was getting annoyed with the large numbers of small pirates that kept coming in and disabling all the other ships in the system and then would come after me. Then I noticed that all of the pirate systems are more than 10 jumps away, way too far for fleets of (even small) pirates to bother coming to raid some miners when they could capture merchants in the South or Core with much less risk. 

Where do these pirates at the western end of the Dirt Belt come from? They couldn't get this far without being blown out of the sky by the authorities, or having to land at some point to refuel. Either there shouldn't be any pirates up here, or there should be a pirate planet nearby where the raids come from. Because Orbona can be a very profitable system early on and Terminus is important to the Remnant, there has to be some risk other than being stranded to go that way. Pirates do that well and so they should stay in that area. But, a new planet is unlikely because new human planets / systems are discouraged, and the area is very close to the Deep (although how pirate ships even get to the Deep isn't answered). It would also draw attention to Terminus which I feel should be an out-of-the-way system. While the manual states that there are pirate cells within poorer planets, there's so many pirates in this little bit of uninhabited human space that this doesn't make sense. 

One idea I just had now is to have a pirate cell on a moon of a gas planet or on an asteroid in one of those uninhabited systems. It wouldn't be visible unless the player went and looked for it. The base directs the raids on this end of the Dirt Belt and up the Deep. However, the base is too far away or totally unknown to the Republic, and anyway it's too small for them to bother; but it could be part of a mission chain for the player to destroy the pirate base up in that area. I haven't thought much more than this, though, I only want to explain how there could be so many pirates in an uninhabited corner of human space.

**Comment by mOctave** [comment]
Okay, that is a very long ways. Even if we presume that they only go through militia-guarded systems while possible, that would still mean pirates with ramscoops and enough heavy warships to draw the attention of the Republic. 

You say new human planets / systems are discouraged: In the 0.9.15 update at least two new human systems were added (Atik and Misam, as well as the pirate world of New Tortuga). New planets and systems in human space are uncommon, but if there's a good reason to add them I don't see why that wouldn't work.

A more likely chance is that pirates are secretly based on the low security worlds nearby (such as Bounty), and are basing their raids from there. Not all pirates have to come from pirate planets, after all.

**Comment by Zitchas** [comment]
I personally assume that most of the smaller pirates are actually opportunistic and/or desperate merchants or miners who normally run under merchant transponder codes and for whatever reason today they just ditch that transponder and decide to see if they can make an easier credit through theft.

These are "small p pirates", the sort who are wanted on charges of stealing a few tons of clothing or pretty theft. They aren't idealistic anarchists or committed Pirates. They aren't capturing ships or looting major outfits or members of the Pirate Fleets, and unless they are actively attacking someone, other merchants and police don't really care about them.

**Comment by bene-dictator** [comment]
Hahaha, I've only just done the Archaeology missions now to find out Watcher is a pirate hideout. Never mind then.
